### PR TITLE
rdm: add August blog post

### PR DESCRIPTION
**Pull request by fenekku**
(rebased)

Notifying @saragon02

### CODE REVIEW

File: content/blog/2020-09-01-august-release/contents.lr [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+---
+author: Sara Gonzales, Guillaume Viger
+---
+body:
+
+We are happy to announce InvenioRDM Alpha 10 (August release)! Thank you to our team members for their efforts on this release.
+
+### What's new?
+
+With less resources than usual, we made some limited changes this month. However, three key areas *were* tackled.
```
Comment (jbenito3) [inline content/blog/2020-09-01-august-release/contents.lr:11]: It's true that there were fewer resources in the InvenioRDM team, but there was an important number of people joining Invenio Sprint, which brought new features to RDM as well :)

**Comment by jbenito3** [comment]
LGTM! thanks @fenekku @saragon02

File: content/blog/2020-09-01-august-release/contents.lr (hunk 1 above)
Comment (fenekku) [reply content/blog/2020-09-01-august-release/contents.lr:11]: But @zzacharo :muscle: was on vacation part of the time: he's a big deal ;)

Comment (jbenito3) [reply content/blog/2020-09-01-august-release/contents.lr:11]: that's true, but if you look at it on the other way, there +10 people working on Invenio this month :)

Comment (fenekku) [reply content/blog/2020-09-01-august-release/contents.lr]: Haha ok, I've removed it :grin:

**Comment by jbenito3** [comment]
merged!

**Pull request merged**